frontend/components/site/sitePublishedBranchesTable.js

```diff
@@ -62,7 +64,7 @@ class SitePublishedBranchesTable extends React.Component {
   }
 
   renderLoadingState() {
-    return <p>No published branch data available</p>
+    return <LoadingIndicator/>
```

Comment:
Are we OK with showing those users the loading indicator? That's a much smaller edge case than the expected flash-of-missing-content-notice case, so this is still better than what we have.